Realm/RLMRealm.h

```diff
@@ -99,6 +99,13 @@
  The on-disk storage for encrypted Realms are encrypted using AES256+HMAC-SHA2,
  but otherwise they behave like normal persisted Realms.
 
+ Encrypted Realms currently cannot be opened while lldb is attached to the
```

Comment:
This bit isn't true anymore (it was true when we used a signal handler), and should instead say something about debugging with encrypted realms leading to lldb hanging. A link to the bug discussing would be nice too.